webpack.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
   web3Loader: {
     provider: web3provider,
     constructorParams: {
-      Chess: []
+      Chess: [true] // Enable debugging
```

Comment:
Is there a way to enable debugging in `npm start`, `npm test`, ... but not in `npm build` or to pass a parameter or env-variable? Atm. I think it will be deployed with debug enabled :/